reactome_analysis_datasets/reactome_analysis_datasets/dataset_searcher/public_data_searcher.py

```diff
@@ -0,0 +1,109 @@
+import json
```

Comment:
I guess this file should be deleted now?

Reply:
file delted